code/modules/surgery/organs/liver.dm

```diff
@@ -64,13 +64,19 @@
 /obj/item/organ/liver/cybernetic
 	name = "cybernetic liver"
 	icon_state = "liver-c"
-	desc = "An electronic device designed to mimic the functions of a human liver. It has no benefits over an organic liver, but is easy to produce."
+	desc = "An electronic device designed to mimic the functions of a human liver. Handles toxins slightly better than an organic liver."
 	synthetic = TRUE
 
+/obj/item/organ/liver/cybernetic/Initialize(mapload)
+	. = ..()
+	maxHealth = round(maxHealth*1.1, 1)
+	toxTolerance = round(toxTolerance*1.1, 1)
+	toxLethality = round(toxLethality*0.9, 0.001)

```

Comment:
why are you setting the vars in Initialize instead of the normal way?

Reply:
So you don't have to check these values whenever you want to edit the normal liver ones.

Reply:
This also affects the upgraded liver, so instead of 200 health it has 220 and so on.

Setting vars on initialize if not necessary is not a good idea imo, it causes issues with subtypes and adds unnecessary processing to creating it.

Reply:
I disagree on the processing for uncommon items such as these but the initialize was oversight so i'll correct it.